RELEASE-NOTE.md

```diff
@@ -1,3 +1,28 @@
+3.5.4 (2021.03.10)
+==================
+- Changes
+    - Change the HttpClient of the test script to apache http components base
+        * Support modern HTTP specification
+        * Provide easy-to-use APIs
+    - Look up controller ip from dns when agent re-tries to connect to controller
```

Comment:
Use controller DNS instead of IP to support agent-controller reconnection in cloud env.